### PR TITLE
Team 101 Days of Summer : Blog Post 2

### PR DESCRIPTION
**Pull request by nupurbaghel**
We have completed our blog post 😄 !! cc @paavininanda

### CODE REVIEW

File: blog/_posts/2018-09-15-101daysofsummer-blog2.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: An enriching journey, Summers’ 2018 :: Team 101 Days of Summer
```
Comment (v-thakkar) [inline blog/_posts/2018-09-15-101daysofsummer-blog2.md]: Hi @nupurbaghel ! Thanks for the PR. Some minor comments here. 

Markdown doesn't seem to be transforming "::" into HTML properly. Please use "-" or any other symbol for the title.

File: blog/_posts/2018-09-15-101daysofsummer-blog2.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+layout: post
+created_at: Sat Sep 15 2018
+permalink: blog/2018-09-15-101daysofsummer-blog2
+current: blog
+author: Nupur Baghel
+twitter: nupur_baghel
+categories: student-posts
```
Comment (v-thakkar) [inline blog/_posts/2018-09-15-101daysofsummer-blog2.md]: To be consistent with other files, Please declare categories in the line below. You can check other posts.

File: blog/_posts/2018-09-15-101daysofsummer-blog2.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+---
+
+## Everything is a dream, we can't stop smiling!!!
+![In our happy workspace (our hostel room :P) with tons of memories from this Summer with Servo and RGSoC ❤️ We wish we had taken a picture with Rakhi and Josh from one of our remote calls :P We’ll take one  soon... ;)](/img/blog/2018/rgsoc-2019-09-15-101daysofsummer-blog2-01.jpeg)
```
Comment (v-thakkar) [inline blog/_posts/2018-09-15-101daysofsummer-blog2.md]: Add image description and use image-credits class to add image credits for this picture. Check the output before pushing the revised commit.

File: blog/_posts/2018-09-15-101daysofsummer-blog2.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+We had to extract Something and use it. All we had to do is dereference, but how? It is very simple.
+```
+b = *a 
+```
+
+And use b directly. * Can be used to dereference as many times as you want.
+
+### Match 
+A very good alternative to if else conditions in Rust. An example is -
+![how to use match example](/img/blog/2018/rgsoc-2019-09-15-101daysofsummer-blog2-02.jpeg)
```
Comment (v-thakkar) [inline blog/_posts/2018-09-15-101daysofsummer-blog2.md]: Add image description and image credits here as well.

File: blog/_posts/2018-09-15-101daysofsummer-blog2.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+```
+
+<hr>
+Couldn’t understand anything of what was said above?
+Here is a very good source to get started with these basic things in Rust - http://intorust.com/
+Every time we faced issues in any of the above, our mentor Josh was there to help us! Not just him, everyone on IRC channel of Servo used to help us in solving these little things. We completely loved the Servo community <3 So feel free to ask anyone anytime you feel like contributing or get stuck somewhere!
+
+
+## Let us get to know about some statistics now :P
+- In total we got **20 PR’s** merged by the end of RGSoC. Approximately 10 each for the both of us. Hooray!!
```
Comment (v-thakkar) [inline blog/_posts/2018-09-15-101daysofsummer-blog2.md]: Do you have single link where people can see all your contributions/commits? I think it'll be good to put that link somewhere here.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2018-09-15-101daysofsummer-blog2.md (hunk 5 above)
Comment (nupurbaghel) [reply blog/_posts/2018-09-15-101daysofsummer-blog2.md]: We have 2 links, one for each of us. I can put that if required..

**Review by v-thakkar — APPROVED** [review]

**Pull request merged**